### PR TITLE
#208 - Support dart retrofit 4.1.0 @Extras parameter option

### PR DESCRIPTION
**Pull request by dfdgsdfg**
#208

### CODE REVIEW

File: swagger_parser/lib/src/generator/model/programming_language.dart [hunk 1]
```diff
@@ -109,6 +109,7 @@ enum ProgrammingLanguage {
     String name, {
     required bool markFilesAsGenerated,
     required String defaultContentType,
+    required bool addExtraParameter,
```
Comment (Carapacik) [inline swagger_parser/lib/src/generator/model/programming_language.dart:112]:
```suggestion
    bool extrasParameterByDefault = false,
```

Comment (Carapacik) [reply swagger_parser/lib/src/generator/model/programming_language.dart:112]: And than fix on line 121

Comment (Carapacik) [reply swagger_parser/lib/src/generator/model/programming_language.dart:112]: Also update version to 1.17.0

**Review by Carapacik — APPROVED** [review]

**Review by Carapacik — APPROVED** [review]
LGTM

**Comment by Carapacik** [comment]
> Hey! I still test this on my production environment. After that, I request merge. : )

Ready for merge

**Comment by dfdgsdfg** [comment]
I tested my production environment. Works pretty well. Ready for merge. 🆗

**Pull request merged**